CoEpi/ble/ContactReceivedHandler.swift

```diff
@@ -0,0 +1,22 @@
+import Dip
+import Foundation
+
+class ContactReceivedHandler: PeripheralRequestHandler {
+
+    private let cenRepo: CENRepo
+    private let cenKeyRepo: CENKeyRepo
+
+    init(container: DependencyContainer) {
+        self.cenRepo = try! container.resolve()
+        self.cenKeyRepo = try! container.resolve()
```

Comment:
Inject specific dependencies. I saw we have this anti-pattern everywhere. I did this for the root wireframe exceptionally because it may need to resolve a lot of dependencies and it becomes impractical, though also discussable.

Reply:
Ah of course, this is obvious I just didn't think about it. Done